.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+  - repo: https://github.com/dnephin/pre-commit-golang
+    rev: v0.5.0
+    hooks:
+    - id: golangci-lint
+    - id: go-unit-tests
+    - id: go-build
+    - id: go-mod-tidy
```

Comment:
When will this be invoked, there are times (when trying to structure atomic commits) that it doesn't actually benefit you to run go-mod-tidy pre-commit

Reply:
I'm fine with removing this.
```suggestion
```